src/main/java/uk/gov/justice/hmpps/datacompliance/events/listeners/DataComplianceEventListener.java

```diff
@@ -44,29 +46,37 @@ public class DataComplianceEventListener {
     private static final String FREE_TEXT_MORATORIUM_RESULT = "DATA_COMPLIANCE_FREE-TEXT-MORATORIUM-RESULT";
     private static final String OFFENDER_RESTRICTION_RESULT = "DATA_COMPLIANCE_OFFENDER-RESTRICTION-RESULT";
     private static final String DECEASED_OFFENDER_DELETION_RESULT = "DATA_COMPLIANCE_DECEASED-OFFENDER-DELETION-RESULT";
+    private static final String OFFENDER_NO_BOOKING_DELETION_RESULT = "DATA_COMPLIANCE_OFFENDER-NO_BOOKING-DELETION-RESULT";
+
```

Comment:
Noticed a couple of these have inconsistent casing, might be worth tidying up at some point. The convention that it was trying to follow was DATA_COMPLIANCE_ (the service) followed by the event name in kebab case. Doesn't hugely matter as long as the producer and listener both agree though!